utils/order-utils/package.json

```diff
@@ -8,7 +8,7 @@
     "url": "https://github.com/airswap/airswap-protocols"
   },
   "scripts": {
-    "test": "mocha test"
+    "test": "mocha test --timeout 3000"
```

Comment:
why is the timeout required?

Reply:
The tests are quite time consuming as they query rinkeby. Normally they run in around 800ms-1400ms. The Mocha default is 2000ms cut off, so I increased this just in case something runs inefficiently.

Reply:
May want to disregard this since there is no `truffle-config.js` in this directory since it's not connecting through that method.